### PR TITLE
Adding WordCounter project to the repository. [Hacktoberfest] 

### PR DESCRIPTION
**Pull request by thevirengarg**
Issue #105 
Adding a front end project to the repo, which calculates typing speed of the user and helps in practicing speed typing.
This website is created using HTML, CSS, javascript.

Screenshot (287)

Screenshot (289)

Screenshot (288)

### CODE REVIEW

**Comment by thevirengarg** [comment]
@VidhiBhatt01 Please review this Pull request.

**Comment by thevirengarg** [comment]
@VidhiBhatt01 Open index.html file in your browser to view the website

File: Projects/WordCounter/stylee.css [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    .boxes {
+        width: 150px;
+    }
+
+    textarea {
+        padding: 50px 0px 0px 30px;
+        width: 300px;
+    }
+}
+
```
Comment (VidhiBhatt01) [inline Projects/WordCounter/stylee.css:201]: nit: Extra line

File: Projects/WordCounter/stylee.css [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+
+.box, .box2 {
+    margin-bottom: 1em;
+
+}
+
+
```
Comment (VidhiBhatt01) [inline Projects/WordCounter/stylee.css:134]: extra line

File: Projects/WordCounter/stylee.css [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    width: 1000px;
+    color: #0e101a;
+    background-color: #ffffff;
+    padding: 50px 30px 80px 30px;
+    border-radius: 8px;
+    box-shadow: 30px 30px 100px rgba(0, 0, 0, 0.3);
+
+}
+
+
```
Comment (VidhiBhatt01) [inline Projects/WordCounter/stylee.css:127]: extra lines

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
Very poor file naming. Rename index3, indexx, stylee files. Follow a uniform naming structure like main1, main2, style1, style2.... or name it something more suitable. 

Clean up your existing files, ident them properly and remove multiple extra lines

**Comment by VidhiBhatt01** [comment]
> @VidhiBhatt01 Open index.html file in your browser to view the website

You should deploy it somewhere and attach the deployed link in the PR description

**Pull request closed without merging**